admin/routes/completed-check.js

```diff
@@ -1,10 +1,19 @@
 const express = require('express')
 const router = express.Router()
+const moment = require('moment')
 const { postCheck } = require('../controllers/completed-check')
+const winstonLogger = require('./../winstonLogger')
 
 router.route('/').all((req, res) => {
-  if (req.method !== 'POST') return res.sendStatus(405)
+  const apiLogger = winstonLogger.apiLogger()
+  apiLogger.info(`MTC-API: completed-started (${moment()}) reached`)
+  if (req.method !== 'POST') {
+    apiLogger.info(`MTC-API: completed-started: req.method is not POST - res.sendStatus(405) - ${JSON.stringify(req.method)} `)
+    return res.sendStatus(405)
+  }
   postCheck(req, res)
+  apiLogger.info(`MTC-API: completed-started: request.body: ${JSON.stringify(req.body)}`)
```

Comment:
we need to accomodate the fact that some requests could be a DOS attack. you will need to snip this at an appropriate length (such as 2MB)